test.sauce-labs.js

```diff
@@ -19,7 +19,7 @@ testSauceLabs({
 	}, {
 		browserName: 'safari',
 		platform: 'OS X 10.13',
-		version: 'latest'
+		version: '11',
```

Comment:
I don’t like config changes like this without understanding the underlying root issue. I’m going to merge this anyway because 1) the site _seems_ to be ok in Safari 12 on my Mac and 2) I want to open a PR without running into this build issue.

@cherifGsoul it would be good to open up an issue with what you know about why this config change had to be made.